packages/core/src/code-evaluation/rebuildGeometryCli.js

```diff
@@ -34,7 +34,10 @@ export const rebuildGeometryCli = (data) => {
   }
 
   // rootModule should contain exported main and getParameterDefinitions functions
-  const rootModule = requireDesignFromModule(mainPath, requireFn)
```

Comment:
Looks like `requireDesignFromModule` is not used anywhere anymore

Reply:
Yeah... more re-organization is required.